desktop/settings/linux.md

```diff
@@ -183,3 +171,21 @@ Use the **Extensions** tab to:
 - **Show Docker Extensions system containers**
 
 For more information about Docker extensions, see [Extensions](../extensions/index.md).
+
+## Features in development
+
+On the **Feature control** tab you can control your settings for **Beta features** and **Experimental features**.
+
+You can also sign up to the [Developer Preview program](https://www.docker.com/community/get-involved/developer-preview/){:target="_blank" rel="noopener" class="_"} from the **Features in development** tab.
+
+### Beta features
+
+{% include beta.md %}
+
+#### Enable containerd
+
+Turns on the experimental containerd image store. This brings new features like namespaces and faster container startup performance by lazy-pulling images.
```

Comment:
iirc we don't support namespaced API yet (cc @rumpl) but we could add a note about Wasm here instead? e.g.

```suggestion
Turns on the experimental containerd image store. This brings new features like faster container startup performance by lazy-pulling images, and the ability to run Wasm applications with Docker.
```

Reply:
cc @sam-thibault